content/docs/user-guide/programming/_index.md

```diff
@@ -41,13 +57,20 @@ Depending on the part of the engine that you want to develop, there are differen
   - [Programmer's Guide to Component Development](/docs/user-guide/programming/components/)
 
 
-### Develop a Gem
+#### Develop a Gem
   
   - [Gem Module system](/docs/user-guide/programming/gems/overview/)
 
   - [Create an O3DE Gem](/docs/user-guide/programming/gems/creating/)
 
 
-### Develop Atom Renderer features
+#### Develop Atom Renderer features
   
   - [Atom Renderer Developer Guide](/docs/atom-guide/dev-guide/)
+
+
+## Related topics
+
+| Topic | Description |
+| --- | --- |
+| [O3DE C++ Coding Standards](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md) | If you're programming features to contibute to [o3de repository](https://github.com/o3de/o3de), follow the coding standards used throughout O3DE's C++ codebase. |
```

Comment:
Nit for continuity: We have a shortcode for the source repo URL that we actively use elsewhere:
```suggestion
| [O3DE C++ Coding Standards](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md) | If you're programming features to contribute to [o3de repository](https://{{< links/o3de-source >}}), follow the coding standards used throughout O3DE's C++ codebase. |
```